src/main/openapi/daam.yaml

```diff
@@ -1021,3 +1066,17 @@ components:
                 format: date-time
       required:
         - entitlements
+    AcceptLicensesCommand:
```

Comment:
important: to avoid clashing names, please rename this class to something else.

Reply:
Renamed it to AcceptTermsCommand since we are referring to it as accept-terms